static/src/javascripts/projects/common/modules/crosswords/crossword.js

```diff
@@ -802,11 +807,13 @@ class Crossword extends Component {
                     clueInFocus={focused}
                     crossword={this}
                 />
+                <div class="below_controls"></div>
                 <Clues
                     clues={this.cluesData()}
                     focussed={focused}
                     setReturnPosition={this.setReturnPosition.bind(this)}
                 />
+                <div>Below Controls</div>
```

Comment:
Whats the purpose of this div? Will it display to users?

Reply:
No it was used for testing ill remove